### PR TITLE
Feature/vectors new method

### PR DESCRIPTION
**Pull request by s3bul**

### CODE REVIEW

File: plugins/include/vector.inc [hunk 1]
```diff
@@ -177,3 +177,29 @@ stock void MakeVectorFromPoints(const float pt1[3], const float pt2[3], float ou
 	output[1] = pt2[1] - pt1[1];
 	output[2] = pt2[2] - pt1[2];
 }
+
+/**
+ * Checking whether the vectors are different
+ *
+ * @param vec1		First vector.
+ * @param vec2		Second vector.
+ * @param mistake	Max mistake.
```
Comment (yedpodtrzitko) [inline plugins/include/vector.inc]: Isn't it called `deviation` in this context usually?
&lt;/bikeshedding&gt;

Comment (s3bul) [inline plugins/include/vector.inc]: You mean the `mistake` is the wrong name?

Comment (yedpodtrzitko) [inline plugins/include/vector.inc]: I don't say it's a mistake (lol), but the more common name is `deviation`.

**Comment by asherkin** [comment]
I don't think this is something we're interested in taking upstream - generally feature additions should be discussed before development either on the bug tracker or the forum.

As a note, the FloatCompare native should not be used directly, it is an internal helper for the relational operators.

There is some good information here on how to implement this function properly when the concern is rounding errors: http://floating-point-gui.de/errors/comparison/

**Pull request closed without merging**